### DISCUSSION TITLE
I'm not eligible for free Github Copilot

### DISCUSSION THREAD

**Discussion by argentinaluiz** [community/community#33337] [Copilot Conversations] [answered]
Hi everyone!

I would like about Github Copilot.

I'm working as a teacher and contributing with TI communities making contents about programming languages, WSL, VSCode, microservices, etc, besides that I made many libraries and projects published in Github for everyone, but my account is not eligible for free Github Copilot.

I think that is not correct because I filled the requirements as well.

Thanks for your support!

**Comment by davecheney** [comment, marked as answer]
@argentinaluiz have you completed the application process here, https://education.github.com/discount_requests/teacher_application. Once you have been accepted you will receive a coupon code you can apply to your github account with will give you access to GitHub Pro and Copilot.

**Reply by argentinaluiz** [reply]
Thanks very much @davecheney ! I already have applied on this process!
